ci/jenkins/test.sh

```diff
@@ -375,6 +376,48 @@ function run_conformance {
     fi
 }
 
+function run_e2e_windows {
+    echo "====== Running Antrea e2e Tests ======"
+    export GO111MODULE=on
+    export GOPATH=${WORKDIR}/go
+    export GOROOT=/usr/local/go
+    export GOCACHE=${WORKDIR}/.cache/go-build
+    export PATH=$GOROOT/bin:$PATH
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    clean_for_windows_install_cni
+    kubectl apply -f build/yamls/antrea.yml
+    kubectl apply -f ${WORKDIR}/kube-proxy-windows.yml
+    kubectl apply -f build/yamls/antrea-windows.yml
+    kubectl rollout restart deployment/coredns -n kube-system
+    kubectl rollout status deployment/coredns -n kube-system
+    kubectl rollout status deployment.apps/antrea-controller -n kube-system
+    kubectl rollout status daemonset/antrea-agent -n kube-system
+    kubectl rollout status daemonset.apps/antrea-agent-windows -n kube-system
+    kubectl rollout status daemonset/kube-proxy-windows -n kube-system
+    kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 != role && $1 ~ /win/ {print $6}' | while read IP; do
+        for i in `seq 5`; do
+            sleep 5
+            timeout 5s ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "powershell Get-NetAdapter -Name br-int -ErrorAction SilentlyContinue" && break
+        done
+        sleep 10
+    done
+
+    mkdir -p test/e2e/infra/vagrant/playbook/kube
+    cp -f "${WORKDIR}/kube.conf" test/e2e/infra/vagrant/playbook/kube/config
+    cp -f "${WORKDIR}/ssh-config" test/e2e/infra/vagrant/ssh-config
```

Comment:
this comes a bit out of nowhere in this context, can we use the `remote` provider instead of the `vagrant` one?
https://github.com/vmware-tanzu/antrea/blob/main/test/e2e/providers/remote.go

Reply:
Ok, let me have a try.

Reply:
I guess your "this" means path like "test/e2e/infra/vagrant/playbook/kube/config"? Updated code to use `remote` provider.